ydb/core/tx/datashard/prefix_kmeans.cpp

```diff
@@ -102,15 +102,19 @@ class TPrefixKMeansScanBase: public TActor<TPrefixKMeansScanBase>, public NTable
 
     ui32 PrefixColumns;
     TSerializedCellVec Prefix;
-    bool HasNextPrefix = false;
+    TBufferData PrefixRows;
```

Comment:
Maybe at least todo about better is storing something like `std::vector<std::pair<TSerializedCellVec, TSerializedCellVec>>` instead of `std::vector<std::pair<TSerializedCellVec, TString>>` to avoid reparsing values

Maybe just template parameter for TBufferData?